package.json

```diff
@@ -1,18 +1,31 @@
 {
   "name": "llnode",
-  "version": "1.4.0",
+  "version": "0.0.0",
   "description": "llnode test suite",
   "main": "no-entry-sorry.js",
   "directories": {
     "test": "test"
   },
+  "//": "We need to use the install scripts as llnode must be built on the machine it's deployed on.",
+  "//": "(Blame C++)",
   "scripts": {
+    "preinstall" : "node scripts/configure.js",
+    "install" : "./gyp_llnode && make -C out/",
```

Comment:
I was trying to leave the build process the same for cloning from git and installing via npm really.